### PR TITLE
Update Splash Screen with warning about openapi docs

### PR DESCRIPTION
**Pull request by eyurtsev**
Update the splash screen to output a warning about OpenAPI docs missing when
using pydantic v2.

### CODE REVIEW

**Comment by eyurtsev** [comment]
Hopefully will help with these issues:

https://github.com/langchain-ai/langserve/issues/214
https://github.com/langchain-ai/langserve/issues/215
https://github.com/langchain-ai/langserve/issues/207

File: langserve/server.py [hunk 1]
```diff
@@ -290,6 +295,15 @@ def green(text):
             print(f'{green("LANGSERVE:")}  └──> {path}/playground/')
             print(f'{green("LANGSERVE:")}')
         print(f'{green("LANGSERVE:")} See all available routes at {app.docs_url}/')
+
+        if _PYDANTIC_MAJOR_VERSION == 2:
+            print()
+            print(f'{orange("OpenAPI Docs:")}: ', end="")
+            print(
+                "Running with pydantic >= 2: OpenAPI docs for "
+                "invoke/batch/stream/stream_log` endpoints will not "
```
Comment (baskaryan) [inline langserve/server.py:304]: will not *be*

File: langserve/server.py [hunk 2]
```diff
@@ -290,6 +295,15 @@ def green(text):
...
+                "generated; but, API endpoints and playground will not be affected."
```
Comment (baskaryan) [inline langserve/server.py:305]: generated, but API ...

**Review by baskaryan — APPROVED** [review]

File: langserve/server.py (hunk 1 above)
Comment (eyurtsev) [inline langserve/server.py:304]:
```suggestion
                "invoke/batch/stream/stream_log` endpoints will not be "
```

**Pull request merged**